src/spec/gocli/integration/network_lifecycle_spec.rb

```diff
@@ -0,0 +1,387 @@
+require_relative '../spec_helper'
```

Comment:
Could you add a test to cover the case of adding a subnet to the network that already exists and will be used by the deployment. According to [the code it should fail](https://github.com/cloudfoundry/bosh/pull/1959/files#diff-de6ac6385f7bc7569a15ef6b5653d4f0R161).

Reply:
Oh this is a case of `updating` the network which we have decided to address later (we have discussed that with the rest of the team and @cppforlife).